### DISCUSSION TITLE
Rückerstattungen

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#422] [Ideas]
Momentan kann man zwar Rückerstattungen (Teilnahmegebühr einer abgesagten Veranstaltung) als negative Zusatzbeträge abbilden. Allerdings scheitert dann jedoch der Abrechnungslauf, da er mit negativen Beträgen nix anfangen kann. Somit muss man Überweisungen manuell anlegen.
Hier wäre es cool, wenn aus negativen Zusatzbeträgen Überweisungen erzeugt werden könnten (Schritt 1) oder diese sogar mit positiven Zusatzbeträgen verrechnet und dann die Lastschrift niedriger ausfallen würde. Dann könnte man nämlich einfach per Rechtsklick in der Mitgliederliste wieder alle Erstattungen auf einmal durchführen.

**Comment by lenilsas** [comment]
Das Verrechnen mit Lastschriften funktioniert bereits, solange insgesamt ein positiver abzubuchender Betrag herauskommt.

**Comment by jokr0815** [comment]
Hallo,
leider ist es aktuell nicht sonderlich intuitiv.
Könnte man nicht eine Option im Kontextmenü anlegen: 
"Gutschrift zur Rechnung erstellen" und ein Gutschriftsformular dazu nutzen / konfigurierbar machen?
Zumindest machen andere Programme das so.
Danke und vG
Jo

**Reply by JohannMaierhofer** [reply]
Ich habe diesen Vorschlag in PR #1289 aufgegriffen.
